projects/canopy/src/lib/carousel/carousel.notes.ts

```diff
@@ -41,4 +41,6 @@ and in your HTML:
 | \`\`headingLevel\`\`  | The heading level of the description: \`\`1\`\`, \`\`2\`\`, \`\`3\`\`, \`\`4\`\`, \`\`5\`\`, \`\`6\`\`   | number  | undefined |   Yes    |
 | \`\`autoPlay\`\`      | Enables auto play mode when set to true.                                                                 | boolean |   false   |   No     |
 | \`\`autoPlayDelay\`\` | Delay time in milliseconds to switch to next slide when autoPlay mode is set to true.                    | number  |   2000    |   No     |
+
+## PlayCarousel and PauseCarousel methods can be exposed by targeting the Carousel with ViewChild decorators to fix the accessibility issues.
```

Comment:
Not sure you have to specify `to fix the accessibility issues` as most people wouldn't know what you are referring to 🤔

Reply:
Good point.